index.html

```diff
@@ -34,20 +42,25 @@ <h3 class="h3">
         </h3>
 
         <div class="section">
-          <div><img src="image/Vector.png" alt="" /></div>
-          <div><img src="image/linkedin.png" alt="" /></div>
-          <div><img src="image/tw.png" alt="" /></div>
-          <div><img src="image/gmail.png" alt="" /></div>
+          <div class="lli"><img src="image/Vector.png" alt="" /></div>
+          <div class="lli"><img src="image/linkedin.png" alt="" /></div>
+          <div class="lli"><img src="image/tw.png" alt="" /></div>
+          <div class="lli"><img src="image/gmail.png" alt="" /></div>
         </div>
       </section>
 
       <section class="body2">
-        <h2 class="s2h1">My Recent Works</h2>
 
+        <div id="sec2">
+        <h2 class="s2h1">My Recent Works</h2>
         <hr class="hr1" />
+      </div>
 
-        <div>
-          <img src="image/placeholder.png" alt="" />
+        <div id="hed2">
+          <img class="none" id="img1" src="image/img1.png" alt="">
+          <img class="none2" src="image/placeholder.png" alt="" />
+          
+          <div id="hed2p2">
           <h3 class="s3h3">Multi-Post Stories</h3>
           <p class="prj1">
             A daily selection of privately personalized reads; no accounts or
```

Comment:
- [ ] As the previous reviewer said kindly implement the text on the p tag as it is not matching with the Figma design also you should reduce the gap between technologies. kindly implement all the changes asked by previous reviewer.